app/users/delete/template.hbs

```diff
@@ -5,6 +5,6 @@
     updateButtonText=updateButtonText }}
   <div class="alert alert-danger">
     <span class="glyphicon glyphicon-warning-sign"></span>
-    Are you sure you wish to delete the user {{model.name}}?
+    {{t 'messages.are_you_sure_delete'}}{{model.name}}?
```

Comment:
@renato04 Quick notice: With ember-i18n we can interpolate values in translation strings (as to move the question mark into the string) like this: https://github.com/cfleschhut/hospitalrun-frontend/commit/0c6dbb9193739089e51e04ffc459b379ab883f3e :)